apispec/utils.py

```diff
@@ -98,4 +98,4 @@ def validate_swagger(spec):
                 stderr=subprocess.STDOUT,
             )
         except subprocess.CalledProcessError as error:
-            raise exceptions.SwaggerError(error.output.decode('utf-8'))
+            raise exceptions.PluginError(error.output.decode('utf-8'))
```

Comment:
Please, go ahead and fix this the way you like it. I have only fixed it so the tests would pass (I don't know how they did pass before).